subprojects/docs/src/docs/userguide/javaLibraryPlugin.xml

```diff
@@ -87,29 +87,24 @@
             <listitem>types exclusively found in internal classes (future versions of Gradle will let you declare which packages belong to the public API)</listitem>
```

Comment:
Reading this again, I am wondering if we should mention (recommend?) that if your want to have a good separation within your own (large) multi-project build, you should organise it into "api" and "implementation" sub-projects.

Maybe not needed for this initial guide though...